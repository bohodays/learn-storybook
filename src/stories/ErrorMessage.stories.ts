import type { Meta, StoryObj } from "@storybook/react-vite";

import { fn } from "storybook/test";
import ErrorMessage from "../components/ErrorMessage";

// Docs 정보
const meta = {
  title: "Text/ErrorMessage", // 경로를 나타냄
  component: ErrorMessage,
  parameters: {
    layout: "centered",
  },
  tags: ["autodocs"],
  argTypes: {
    children: { control: "text", description: "ErrorMessage의 내용" },
  },
} satisfies Meta<typeof ErrorMessage>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {
    children: "에러 메세지는 여기로",
  },
};
