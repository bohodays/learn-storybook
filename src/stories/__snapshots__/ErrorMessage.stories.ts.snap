// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text/ErrorMessage Default smoke-test 1`] = `
<p class="text-xs text-error">
  에러 메세지는 여기로
</p>
`;
